client/components/LearnerViewDirective.js

```diff
@@ -214,6 +218,11 @@ tie.directive('learnerView', [function() {
           float: left;
           margin-top: 10px;
         }
+        .tie-code-reset {
```

Comment:
If it doesn't break the CSS, please insert alphabetically (after .tie-arrow-highlighter)